org.free_astro.ser_player.json

```diff
@@ -0,0 +1,33 @@
+{
+	"id": "org.free_astro.ser_player",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "6.8",
+	"sdk": "org.kde.Sdk",
+	"command": "ser-player",
+	"cleanup": [
+		"/include",
+		"*.a",
+		"*.la",
+		"*.cmake"
+	],
+	"finish-args":
+	[
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [
+		{
+			"name": "ser-player",
+			"buildsystem": "qmake",
+			"config-opts": [],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/nouspiro/ser-player.git",
```

Comment:
why pulling from a fork. original seems to be https://github.com/lock042/ser-player

Reply:
It is not merged yet. I will change that to original when it is merged.

Reply:
at least getting the `flatpak` branch merged there.

Reply:
Ok branch was merged. I changed url to original lock repository.